concepticondata/conceptlists/Hartmann-2013-162.tsv

```diff
@@ -1,163 +1,163 @@
+Hartmann-2013-162-107	107	MAKE RUN						The girl made the horse run.	A makes P run	New meaning	make-run
+Hartmann-2013-162-108	108	MAKE SICK						The cold made the girl sick.	M makes E sick	New meaning	make-sick
+Hartmann-2013-162-109	109	MARRY				203	MARRY	The man married the woman.	X marries Y	New meaning	marry
+Hartmann-2013-162-110	110	MEET	vstretitʹ	treffen	meet	1831	MEET	The men met the boys.	A meets X	Core list	meet
+Hartmann-2013-162-111	111	NAME	nazvatʹ	nennen	call OR name	1405	NAME	The parents called the baby Anna.	A name X (a) Y	Core list	name
+Hartmann-2013-162-112	112	OFFER						The girl offerd her help to a stranger.	X offers Y to Z	New meaning	offer
+Hartmann-2013-162-113	113	PEEL	očistitʹ	schälen	peel	275	PEEL	The boy peeled the bark off the stick.	A peels (X off) P	Core list	peel
+Hartmann-2013-162-114	114	PLAY	igratʹ (1) OR igratʹ (2)	spielen	play	1413	PLAY	The child is playing.	S plays	Core list	play
+Hartmann-2013-162-115	115	POUR	litʹ	gießen	pour	1487	POUR	The man poured water into the glass.	A pours T somewhere (L)	Core list	pour
+Hartmann-2013-162-116	116	PROPOSE (a marriage)						The man proposed to the woman.	A proposes to M	New meaning	propose-a-marriage
+Hartmann-2013-162-117	117	PUSH	tolknutʹ	stoßen OR schieben OR schubsen	push	1452	PUSH	The boy pushed the girl (into the water).	A pushes P (somewhere (L))	Extended list	push
+Hartmann-2013-162-118	118	PUT	položitʹ	stellen OR legen OR setzen	put	998	PUT	I put the cup onto the table.	A puts T somewhere (L)	Core list	put
+Hartmann-2013-162-119	119	RAIN	dožd' idët	regnen	rain	1253	RAIN (RAINING)	It rained.	(it) rains	Core list	rain
+Hartmann-2013-162-120	120	READ				724	READ	The girl is reading a book.	X reads Y	New meaning	read
+Hartmann-2013-162-121	121	REFUSE				49	REFUSE	The old lady refused the man a place to sleep.	X refuses Y Z	New meaning	refuse
+Hartmann-2013-162-122	122	REMAIN				987	REMAIN	The woman remained in the village.	A remains in L	New meaning	remain
+Hartmann-2013-162-123	123	REMEMBER	zapomnitʹ			707	REMEMBER	She remembered the street.	A remembers X	Old meaning	remember
+Hartmann-2013-162-124	124	REST				168	REST	The girl was resting.	X rests	New meaning	rest
+Hartmann-2013-162-125	125	ROLL	katitʹsja	rollen	roll	827	ROLL	The ball is rolling.	A rolls	Core list	roll
+Hartmann-2013-162-126	126	ROLL (tr)						The child rolls the ball.	A rolls P	New meaning	roll-tr
+Hartmann-2013-162-127	127	RUN	bežatʹ	rennen	run	1519	RUN	The horse is running.	A runs	Core list	run
+Hartmann-2013-162-128	128	SAY	skazatʹ OR govoritʹ	sagen	say	1458	SAY	"They said ""no"" to me."	"A says ""...""( to X)"	Core list	say
+Hartmann-2013-162-129	129	SAY (impersonal)						It says (it) in the article.	S says somewhere (L)	New meaning	say-impersonal
+Hartmann-2013-162-130	130	SCREAM	vopit'	schreien	scream	2522	SCREAMING	The man screamed.	S screams	Core list	scream
```

Comment:
Judging by the ontological category of SCREAMING, it's rather a noun. So I'd leave this one unmapped.

Reply:
We need to have a look at the other mappings (https://concepticon.clld.org/parameters/2522) then as well.